docs/tutorials/jax-export.ipynb

```diff
@@ -1,445 +1,479 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true,
+    "id": "KZEd7NavBoem",
+    "jupyter": {
+     "outputs_hidden": true
+    }
+   },
+   "outputs": [],
+   "source": [
+    "!pip install tensorflow-cpu"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "lf7Fnsrop7BD"
+   },
+   "source": [
+    "### Export to SavedModel using `jax2tf`\n",
+    "\n",
+    "JAX provides a simple API for exporting StableHLO into a format that can be packaged in SavedModel in `jax.experimental.jax2tf`. This uses the `export` function under the hood, so the same `jit` requirements apply.\n",
+    "\n",
+    "Full details on `jax2tf` can be found in the [README](https://github.com/google/jax/blob/main/jax/experimental/jax2tf/README.md#jax-and-tensorflow-interoperation-jax2tfcall_tf). For this example, we'll only need to know the `polymorphic_shapes` option to specify our dynamic batch dimension."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
     },
+    "id": "GXkgtX7QEiWa",
+    "outputId": "9034836e-4ba1-4210-c2c1-316777d4ad89"
+   },
+   "outputs": [
     {
-      "cell_type": "markdown",
-      "source": [
-        "### Reload and call the SavedModel\n",
-        "\n",
-        "Now we can load that SavedModel and compile using our `sample_input` from a previous example.\n",
-        "\n",
-        "_Note: the restored model does *not* require JAX to run, just XLA._"
-      ],
-      "metadata": {
-        "id": "CmKABmLdrS3C"
-      }
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[34massets\u001b[m\u001b[m         fingerprint.pb saved_model.pb \u001b[34mvariables\u001b[m\u001b[m\n"
+     ]
     },
     {
-      "cell_type": "code",
-      "source": [
-        "restored_model = tf.saved_model.load('/tmp/resnet18_tf')\n",
-        "restored_result = restored_model.f(tf.constant(sample_input, tf.float32))\n",
-        "print(\"Result shape:\", restored_result[0].shape)"
-      ],
-      "metadata": {
-        "colab": {
-          "base_uri": "https://localhost:8080/"
-        },
-        "id": "9Az3dXXWrVDM",
-        "outputId": "cac6d3b1-126e-4e66-dc4d-f2a798ede463"
-      },
-      "execution_count": 8,
-      "outputs": [
-        {
-          "output_type": "stream",
-          "name": "stdout",
-          "text": [
-            "Result shape: (1, 512, 7, 7)\n"
-          ]
-        }
-      ]
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/pavithraes/mambaforge/envs/shlo-docs/lib/python3.11/pty.py:89: RuntimeWarning: os.fork() was called. os.fork() is incompatible with multithreaded code, and JAX is multithreaded, so this will likely lead to a deadlock.\n",
```

Comment:
<img width="1086" alt="image" src="https://github.com/user-attachments/assets/7827045f-0421-4110-ace2-50824b8cc3bf">

Can we see if there's a way to suppress errors? Not sure if there's a way to write this code without some implicit call to fork, but don't want to show your user path here 🙂

Reply:
Oh right, I was meaning to clean this but missed it. Will do!